can/interfaces/pcan/basic.py

```diff
@@ -14,6 +14,7 @@
 from string import *
 import platform
 import logging
+import winreg
```

Comment:
Do you mind reverting these changes in `basic.py` to detect the registry key? Let's keep this PR's scope to adding support for error frames.

Reply:
Ok will do. I make a new PR or it's not needed in the package?